apps/frontend/src/libs/components/table-pagination/styles.module.css

```diff
@@ -0,0 +1,38 @@
+.container,
```

Comment:
define these styles separately, and add them only where they are really needed. The same for styles below